### ISSUE TITLE
find an error

### ISSUE THREAD

**Issue by zyj183247166** [vijayvee/Recursive-neural-networks-TensorFlow#2]
![image](https://user-images.githubusercontent.com/7501074/48712180-16419f80-ec48-11e8-8492-da142736fd6c.png)
i think one line of code "new_model=True" should be added in it just as the picture shows.

**Comment by zyj183247166** [comment]
no, i make an error. one line of code "new_model=False" should be added in the place just as the picture shows.

**Comment by guangyuanyu** [comment]
good catch
